app/services/websites/create.rb

```diff
@@ -14,7 +14,15 @@ def initialize(params, user)
     end
 
     def call
-      if params[:website][:url].to_s.include? '.'
+      already_exist = Website.find_by(url: params[:website][:url])
```

Comment:
`already_exist` does not convey what object we're fetching here. Can you think of a better name for it?